db/db_impl/db_impl.cc

```diff
@@ -1487,12 +1487,15 @@ void DBImpl::MarkLogsSynced(uint64_t up_to, bool synced_dir,
   for (auto it = logs_.begin(); it != logs_.end() && it->number <= up_to;) {
```

Comment:
Another idea from @rishabh, Can you print the last sequence number, when closing a wal file. ie when a memtable/wal is swapped.